lib/active_admin/error_pages/error_pages.rb

```diff
@@ -0,0 +1,9 @@
+ActiveAdmin.after_load do |app|
+  app.namespaces.each do |namespace|
+    namespace.register_page "Error" do
+      menu false
+
+      content title: proc { "Error #{env["active_admin.original_exception"].status_code}" }
```

Comment:
Line is too long. [91/80]<br>Prefer single-quoted strings inside interpolations.